tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -833,6 +833,31 @@ public static function data_html_with_unbalanced_tags() {
 		);
 	}
 
+	/**
+	 * Tests that the `process_directives` handles self-closing BR tags without
+	 * causing fatal errors and processes directives correctly.
+	 *
+	 * @ticket 63891
+	 * @covers ::process_directives
+	 */
+	public function test_process_directives_handles_br_self_closing_tags_with_invalid_closers() {
+		$this->interactivity->state(
+			'myPlugin',
+			array(
+				'id' => 'some-id',
+			),
```

Comment:
I think this is the problem, PHP 7.2 doesn't support trailing commas in function arguments:
```suggestion
			)
```

Reply:
oh! it was that! 
Too much time using PHP 8+ 😅 I didn't noticed
Thanks